src/tokenize.cpp

```diff
@@ -1902,6 +1912,18 @@ void tokenize(const deque<int> &data, chunk_t *ref)
          cpd.preproc_ncnl_count = 0;
       }
 
+      /* Enable/disable indentation between #asm / #endasm diretives */
+      if (pc->type == CT_PP_ASM )
+      {
+         LOG_FMT(LBCTRL, "Found a directive %s on line %d\n","#asm",pc->orig_line);
+         cpd.unc_off = true;
```

Comment:
Disabling the entire processing would be very an extreme approach. Uncrustify already supports formatting of the assembly code. Wouldn't that be a better approach ?